### DISCUSSION TITLE
KeyVault events after renew

### DISCUSSION THREAD

**Discussion by lassi-pesari-qoco** [polymind-inc/acmebot#588] [Q&A] [answered]
Why Microsoft.KeyVault.CertificateNewVersionCreated event not fired after renewing certificate? 

I need to copy a new certificate version to another keyvault automatically using Azure Function triggered by EventGridTrigger Microsoft.KeyVault.CertificateNewVersionCreated, but it never fired after hitting "Renew" button on certificate details pop-up. 

If I generate a new version for some certificate manually, event is fired.

**Comment by shibayan** [comment, marked as answer]
Acmebot is only issuing certificates as non-partnered CAs for Key Vault, so it is not involved with Event Grid. We recommend that you create a support request.

**Reply by lassi-pesari-qoco** [reply]
> Acmebot is only issuing certificates as non-partnered CAs for Key Vault, so it is not involved with Event Grid. We recommend that you create a support request.

What is meant by "support request"? Also I assumed that an event is fired automatically by Azure when new cetificate version is generated to keyvault.

**Reply by shibayan** [reply]
Contact Azure support, it is out of Acmebot's domain.
